packages/contractkit/src/wallets/signers/ledger-signer.ts

```diff
@@ -169,6 +170,15 @@ export class LedgerSigner implements Signer {
       if (tokenInfo) {
         await this.ledger!.provideERC20TokenInformation(tokenInfo)
       }
+      if (rlpEncoded.transaction.feeCurrency) {
```

Comment:
this is a sanitised value, so you should check against `0x`
replace it for this line:
`if (rlpEncoded.transaction.feeCurrency && rlpEncoded.transaction.feeCurrency !== '0x') {`

Reply:
Oh, thanks!